overrides/config-overrides/normal/server.properties

```diff
@@ -0,0 +1,32 @@
+#Minecraft server properties
+op-permission-level=4
+level-name=world
+allow-flight=true
+prevent-proxy-connections=false
+server-port=25565
```

Comment:
Will this and server.ip be overrided automatically?

Reply:
Because of how server.properties is copied, it will just replace the entire file. Figured it was somewhat of a non-issue since people running mode-switcher are either setting up a server for the first time, or migrating (will replace config later)

Reply:
So, when running the server, those fields will be overrided?

Reply:
It just replaces server.properties when the script is run, the only "real" difference is difficulty=0 instead of difficulty=1, assuming someone has not edited their configs already.